### PR TITLE
[11.0] sale_proforma_report Incorporated into standard in sale

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Pro-forma invoices can be printed from SO after activating a setting.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza I'm don't see clear how to automate the activation (in premigration I don't have the new group in post-migration I don't have the old module) of the group

**Comment by pedrobaeza** [comment]
You can detect if the column `proforma` is present on the table `sale_order`, and then activate the group if so.

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza Thanks for the :bulb:! Done.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]

File: addons/sale/migrations/11.0.1.1/post-migration.py [hunk 1]
```diff
@@ -8,9 +9,20 @@
...
 
 
+def activate_proforma(env):
+    """If module `sale_proforma_report` (OCA/sale-reporting) was installed we
+    need to activate the "Pro-forma Invoices" setting as the module was
+    deprecated in favour of it."""
+    if openupgrade.column_exists(env.cr, 'sale_order', 'proforma'):
+        employee_group = env.ref('base.module_category_human_resources')
+        proforma_group = env.ref('sale.group_proforma_sales')
+        employee_group.write({'implied_ids': [(4, proforma_group.id)]})
```
Comment (pedrobaeza) [inline addons/sale/migrations/11.0.1.1/post-migration.py:19]: Is this the way that Odoo uses when you mark the field in settings?

Comment (LoisRForgeFlow) [reply addons/sale/migrations/11.0.1.1/post-migration.py:19]: yes, exactly, I took it from res.settings.

Comment (LoisRForgeFlow) [reply addons/sale/migrations/11.0.1.1/post-migration.py:19]: btw, I already tested it and worked :green_heart:

**Pull request merged**